.travis.yml

```diff
@@ -4,3 +4,6 @@ node_js:
   - 'stable'
   - '0.12'
   - '0.10'
+env:
+  global:
+    - NPM_TOKEN=00000000-0000-0000-0000-000000000000
```

Comment:
Can you remove this? As discussed in https://github.com/homerjam/package-json/commit/b65cd662046948b8dbd911bddac65ba3732fd09d